packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/TransactionsCompletedRow.scss

```diff
@@ -1,12 +1,19 @@
 .wallets-transactions-completed-row {
+    align-items: center;
     display: flex;
-    padding: 1.6rem;
     justify-content: space-between;
-    align-items: center;
-
+    padding: 1.6rem;
+    width: 100%;
+    min-height: 7.2rem;
     @include mobile {
         padding: 1.6rem 0;
     }
+    @media (hover: hover) {
```

Comment:
is this necessary?

Reply:
Yeap @aizad-deriv , otherwise the hover effect persists on mobile

Reply:
https://github.com/user-attachments/assets/e09b452e-76f5-4865-87ee-865d2320e9be